README.md

```diff
@@ -126,7 +127,15 @@ Under "*outputLocation*":
 * "*includeColumnNames*" boolean (defaults to false) - should be "true" to include a row of column names at the top of the csv file for each table, and "false" for no header row.
 * "*saveAsSingleFile*" boolean (defaults to false) - should be "true" for writing out a single file (.csv or .parquet) per table, and "false" to have multiple/smaller files be written based on SPARK partitioning.
 * "*saveIntoTimestampDirectory*" boolean (defaults to false) - should be "true" to save the CSV files into a directory with savepoint timestamp (/outputLocation/path/table/timestamp/*.csv), and "false" to save directly into the table directory (/outputLocation/path/table/*.csv).
-
+* "*largeTextFields*" is a comma delimited list of table.column values that must allow max length varchar types.  
```

Comment:
This section will conflict with #2, so you should talk to Sasi before merging (or she should talk to you, whatever works).

Reply:
Yes - I've made a few changes to the code since the #2 PR. I'll look at that version and merge them in my master branch (the PR source). I'll reach out to Sasi if I had questions.
I should be able to get to that today.

Reply:
Worked with Sasi - finalizing changes across both PRs with her.